### PR TITLE
feat(github): create release-connect-npm action

### PR DESCRIPTION
**Pull request by karliatto**
## Description
Create an action instead of a template to be used directly.

### CODE REVIEW

File: .github/workflows/release-connect-npm-dependencies.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           submodules: recursive
 
       - name: Deploy to NPM ${{ github.event.inputs.package }}
-        uses: ./.github/workflows/template-release-connect-npm.yml
+        uses: ./.github/actions/release-connect-npm
```
Comment (vdovhanych) [inline .github/workflows/release-connect-npm-dependencies.yml]: this should be `uses: ./.github/actions/release-connect-npm@develop` for us

File: .github/workflows/release-connect-npm.yml [hunk 2]
```diff
@@ -21,7 +21,7 @@ jobs:
           submodules: recursive
 
       - name: Deploy to NPM ${{ github.event.inputs.package }}
-        uses: ./.github/workflows/template-release-connect-npm.yml
+        uses: ./.github/actions/release-connect-npm
```
Comment (vdovhanych) [inline .github/workflows/release-connect-npm.yml]: `uses: ./.github/actions/release-connect-npm@develop`

**Review by vdovhanych — CHANGES_REQUESTED** [review]

File: .github/workflows/release-connect-npm-dependencies.yml (hunk 1 above)
Comment (karliatto) [reply .github/workflows/release-connect-npm-dependencies.yml]: Done https://github.com/trezor/trezor-suite/pull/12103/files#diff-7350b81fd8fe25cd59872c938153a0d92081665fe4e6ee69e63ab0a7a69e78d5R37

File: .github/workflows/release-connect-npm.yml (hunk 2 above)
Comment (karliatto) [reply .github/workflows/release-connect-npm.yml]: Done https://github.com/trezor/trezor-suite/pull/12103/files#diff-480abbafbd688337f51c1c23414e6a5975c65d7d4fb1423261622235c65d5d7fR24

**Review by vdovhanych — APPROVED** [review]
lets see if that works :)

**Pull request merged**